packages/design-system-docs/src/pages/components/ChoiceList/Choice.example.jsx

```diff
@@ -16,13 +16,92 @@ const dropdownOptions = [
 const childDropdown = (
   <Dropdown
     options={dropdownOptions}
-    defaultValue={'B'}
+    defaultValue="B"
     label="Dropdown example"
     labelClassName="ds-c-label ds-u-margin-top--0"
     name="dropdown_choices_field"
   />
 );
 
+class ControlledChoiceButton extends React.PureComponent {
```

Comment:
I appreciate that you included this controlled example, especially for testing this makes things a lot easier, but I think after all the PR feedback is in we should remove this from the example

Reply:
Sounds good, after testing, we can revert the Choice.example.jsx back to the original